spirv_msl.hpp

```diff
@@ -63,7 +63,7 @@ using MSLStructMemberKey = uint64_t;
 
 // Special constant used in a MSLResourceBinding desc_set
 // element to indicate the bindings for the push constants.
-static const uint32_t kPushConstDescSet = UINT32_MAX;
+static const uint32_t kPushConstDescSet = std::numeric_limits<uint32_t>::max();
```

Comment:
Build failure on Linux:
```
spirv_msl.hpp:66:43: error: ‘numeric_limits’ is not a member of ‘std’
 static const uint32_t kPushConstDescSet = std::numeric_limits<uint32_t>::max();
                                           ^
spirv_msl.hpp:66:71: error: expected primary-expression before ‘>’ token
 static const uint32_t kPushConstDescSet = std::numeric_limits<uint32_t>::max();
                                                                       ^
spirv_msl.hpp:66:72: error: ‘::max’ has not been declared
 static const uint32_t kPushConstDescSet = std::numeric_limits<uint32_t>::max();
```

Reply:
Oh-oh!

AFAIK...this is standard C++11 (actually C++98) [functionality](http://www.cplusplus.com/reference/limits/numeric_limits/).

Can someone with Linux C++ experience weigh in here, please?

Reply:
You need <limits>, yes, that should do it.